i18n/en.json

```diff
@@ -43,7 +43,7 @@
 	"requestwiki-label-siteurl": "Subdomain:",
 	"requestwiki-notloggedin": "Sorry, you first need to <span class=\"plainlinks\">[$1 log in]</span> before you can request a wiki.",
 	"requestwiki-submit": "Submit",
-	"requestwiki-success": "Your wiki request ($1) was successfully submitted.",
+	"requestwiki-success": "Your wiki request ($1) was successfully submitted. Please note that it will be verified by a human to prevent spam, this shouldn't take more than 48 hours.",
```

Comment:
Please rewrite the sentence to:
"Your request will be reviewed by a volunteer as soon as possible."